cfpb/consumerfinance.gov
# P4C: Replace link emailing with copying functionality

The grad path tool email link functionality can be problematic for users, so this replaces it with a copy button that simplifies copying your link to the clipboard.


## Additions

- Copy button and code


## Removals

- Email link 





## How to test this PR

1. Check out branch and run gulp
2. Navigate to save and finish page in grad path tool
3. Verify that clicking (or using keyboard to select) the copy button copies the link to the clipboard & that success text is displayed on the copy button

## Screenshots

### Before copying
![791a1280-961a-11eb-929e-e09844a98998](https://user-images.githubusercontent.com/778171/114349588-f82cb080-9b1c-11eb-94c1-ee3bd946690a.png)

### After copying
![715a6e00-961a-11eb-9d34-9b53539344ac](https://user-images.githubusercontent.com/778171/114349564-f06d0c00-9b1c-11eb-8c09-b5dc2f4ac1cd.png)




## Checklist


  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".


### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
